hack/gke/xfs-formatter/xfs-formatter.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euExo pipefail
+shopt -s inherit_errexit
+
+ephemeral_mountpoint='/mnt/stateful_partition/kube-ephemeral-ssd'
+
+raid_device="$( head -n 1 "${RAID_DEVICE_FILE}" )"
+
+systemctl stop kube-node-configuration.service
+systemctl stop kubelet.service
+crictl_pods="$( /home/kubernetes/bin/crictl pods -q )"
+/home/kubernetes/bin/crictl stopp ${crictl_pods}
```

Comment:
nit: quotes

Reply:
unquoted on purpose, there's multiple pods

Reply:
you could use the array expansion but let's keep this as podIDs are unlikely to have spaces

Reply:
but we need to check for the case where there is no pod